application/services/bi.py

```diff
@@ -10,3 +13,16 @@ def host(self) -> str:
 
     def port(self) -> int:
         return 80
+
+    def url_env_var(self) -> str:
+     return "BI_SERVICE_URL"
+
+    def atualizarDashTwitter(self, ids: dict):
+        
+        url: str = f"{self.base_url}/atualizarDashTwitter"
+
+        response: requests.Response = requests.get(url, json=ids)
+        response.raise_for_status()
+
+        return response.json()
```

Comment:
and here you can first build a `TwitterDashboardData` object with the content retrieved from the endpoint just to validate the structure before return it.

```suggestion
        twitter_dashboard_data: TwitterDashboardData = TwitterDashboardData.parse_obj(response.json())
        return twitter_dashboard_data
```

> If you don't receive an expected format (defined on the `TwitterDashboardData` model), a `ValidationError` will be raised.